### PR TITLE
Blog post on restrict authorization

### PR DESCRIPTION
**Pull request by anuragpaliwal80**
Adding blog post on restrict authorization feature of chaos mesh.

### CODE REVIEW

**Comment by milasuperstar** [comment]
This blog is not 100% ready yet, please do not merge :)

File: blog/2021-07-07-restrict-authorization.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+image: /img/chaos-engineering-tools-as-a-service.jpeg
+tags: [Chaos Mesh, Chaos Engineering]
+---
+
+![Chaos engineering tools](/img/chaos-engineering-tools-as-a-service.jpeg)
+
+# Cluster multi-tenancy
+
+A [multi-tenant](https://cloud.google.com/kubernetes-engine/docs/concepts/multitenancy-overview) cluster is shared by multiple users and/or workloads which are referred to as "tenants".The operators of multi-tenant clusters must isolate tenants from each other to minimize the damage that a compromised or malicious tenant can do to the cluster and other tenants.
+When you plan a multi-tenant architecture, you should consider the layers of resource isolation in Kubernetes: cluster, namespace, node, Pod, and container.
```
Comment (milasuperstar) [inline blog/2021-07-07-restrict-authorization.md:17]:
```suggestion
When you plan a multi-tenant architecture, you should consider the layers of resource isolation in Kubernetes: cluster, namespace, node, Pod, and container.

<!--truncate-->

```

Comment (anuragpaliwal80) [reply blog/2021-07-07-restrict-authorization.md:17]: done

**Comment by anuragpaliwal80** [comment]
Can someone please fix the link not found issue in chaos as a service blog post:

```
FILE: ./blog/2021-06-25-chaos-mesh-remake-one-step-closer-toward-chaos-as-a-service.md
[✖] https://vector.dev/docs/reference/transforms/wasm/

34 links checked.

ERROR: 1 dead links found!
```

**Comment by milasuperstar** [comment]
> Can someone please fix the link not found issue in chaos as a service blog post:
> 
> ```
> FILE: ./blog/2021-06-25-chaos-mesh-remake-one-step-closer-toward-chaos-as-a-service.md
> [✖] https://vector.dev/docs/reference/transforms/wasm/
> 
> 34 links checked.
> 
> ERROR: 1 dead links found!
> ```

will take a look

**Comment by anuragpaliwal80** [comment]
FYI... PR: https://github.com/chaos-mesh/website/pull/78 fixing broken link issue.

**Comment by milasuperstar** [comment]
LGTM, @g1eny0ung PTAL 👀

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Pull request merged**